.circleci/config.yml

```diff
@@ -541,11 +541,15 @@ jobs:
           name: Install boto3
           command: sudo pip3 install boto3
       - run:
-          name: Clone spm github repo
+          name: Clone spm github repo and reset release branch
           command: |
             git clone "https://${GITHUB_SPM_RELEASE_TOKEN}@github.com/aws-amplify/aws-sdk-ios-spm.git"
             cd aws-sdk-ios-spm
             git checkout -b pre-release-${release_version}
+            git fetch
+            git checkout release
+            git reset --hard main
+            git push -f
```

Comment:
When would we expect this to be necessary? Why is this a better solution than merging from `main` into `release`?

Reply:
When we make some change directly to the main branch, release will be out of sync. I thought it will be safe to just force reset the release to main branch and not worry about any merge conflicts.

Reply:
Fair enough